features/step-definitions/deleting_letters.steps.js

```diff
@@ -0,0 +1,28 @@
+const { When, Then } = require("@wdio/cucumber-framework");
+const { expect } = require("chai");
+const SentPage = require("../pageobjects/sent.page");
+const sentPage = new SentPage();
+
+When(/^I select all letters$/, async () => {
+  await sentPage.selectAllCheckBox.click();
+});
+
+When(/^I click on Delete button$/, async () => {
+  await sentPage.deletePermanentlyButton.click();
+});
+
+When(/^I confirm deletion$/, async () => {
+  await sentPage.deletePermanentlySubmitButton.waitForDisplayed();
+  await sentPage.deletePermanentlySubmitButton.click();
+});
+
+Then(
+  /^I expect to see no more letters are present inside Sent folder$/,
+  async () => {
+    await sentPage.letterStatusInFolders.waitForDisplayed();
+
+    expect(await sentPage.letterStatusInFolders.getText()).to.equal(
+      "No messages found"
+    );
```

Comment:
you can make your step more universal
e.g.
Then(
  /^I expect "No messages found" text is displayed inside Sent folder$/,
  async (text) => {
    await sentPage.letterStatusInFolders.waitForDisplayed();

    expect(await sentPage.letterStatusInFolders.getText()).to.equal(
      text
    );

it will allow you to use the same step multiple times with different teat